lms/djangoapps/teams/tests/test_models.py

```diff
@@ -124,6 +124,20 @@ def test_add_user_bad_team_access(self):
         with self.assertRaises(AddToIncompatibleTeamError):
             self.masters_team.add_user(self.audit_learner)
 
+    def test_defaults(self):
+        field = CourseTeam._meta.get_field('last_activity_at')
+        default_date = datetime(1978, 1, 1)
+        mock_now = lambda: default_date
+        with patch.object(field, 'default', new=mock_now):
+            team = CourseTeam.create(
+                name='foo',
+                course_id='sedx/the-course/1',
+                description='Import from csv',
+                topic_id=TEAMSET_1_ID,
+                organization_protected=True
+            )
+        self.assertEqual(default_date, team.last_activity_at)
```

Comment:
Nit: also assert that `country` and `language` are the empty string.

Reply:
will do